app/views/content/landing/how-to-fund-your-teacher-training/_content.html.erb

```diff
@@ -99,9 +99,9 @@
       <p>You could also be eligible for up to:</p>
 
       <ul>
-        <li>£331.95 a week to support childcare costs for 2 or more children</li>
+        <li><%= v :finance_childcaregrantweeklypayment_twoormorechildren %> a week to support childcare costs for 2 or more children</li>
         <li><%= v :finance_adultdependantsgrant_value %> a year if an adult depends on you</li>
```

Comment:
This isn't showing as a figure in the left column, but showing ok in the review app